cmd/kyma/alpha/initcmd/module/module.go

```diff
@@ -0,0 +1,73 @@
+package module
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/pkg/module/initcmd"
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new Kyma CLI command
+func NewCmd(o *Options) *cobra.Command {
+
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "module MODULE_NAME <PARENT_DIR>",
+		Short: "Initializes an empty module with the given name in the given parent directory.",
+		Long: `Use this command to create an empty Kyma module with the given name in a correspondingly named subdirectory of provided parent directory.
+
+### Detailed description
+
+Kyma modules are individual components that can be deployed into a Kyma runtime. 
+With this command, you can initialize an empty module folder's for the purpose of further development.
```

Comment:
Not sure about the "folder's" but it seems a bit weird. LMK if my suggestion is misunderstanding the meaning:

```suggestion
With this command, you can initialize an empty module folder for the purpose of further development.
```